recipes/gmail/webview.js

```diff
@@ -16,7 +16,7 @@ module.exports = Ferdium => {
 
   const getMessages = () => {
     let count = 0;
-
+    let count2 = 0;
```

Comment:
Can we name both of these accordingly? I suggest those:
`count -> countImportant`
`count2 -> countNonImportant`